### ISSUE TITLE
kp_slash doesn't work

### ISSUE THREAD

**Issue by Gelmit** [jessewebb/csgo-buynds#5]
For example `bind "kp_slash" "buy p250;"` doesn't work but somehow `bind "-" "buy p250;"` works.
This binds it to numpad / and your actual - key.
Some discussion about this topic can be found here http://www.pwnsource.com/tickets/kp_slash-slash-key-on-the-numpad-is-without-use-bindings-on-that-key-will-not-work

**Comment by SirBalefire** [comment]
I think this is not relly a CSGO-Buynds issue.
"kp_slash" will work fine on US (and proberbly other) keyboard layouts.

It won't work on othters - I'm also trying to find a sloution for this problem.
http://redd.it/3f826h

I think it would be nice if this issue was documented in CSGO-Buynds somehow.

**Comment by jessewebb** [comment]
I agree, this is not necessarily a bug with csgo-buynds. "kp_slash" works for some people, but not others. I am not sure how to detect which bind to use automatically. And this website is far from being internationalized yet (although it isn't out of the question).

For now, I do not want to change the current behavior. It sucks it doesn't work for everyone. But I will add somethign to the site which explains this known issue.

Thanks for reporting/contributing!

**Comment by OMG-Abaddon** [comment]
I noticed this issue was still open, add a dropdown to select your keyboard region, then map special keys for each region.

E.g.:
- default.kp_slash = "kp_slash"
- spanish.kp_slash = "-"

That way you bind it correctly in any case. The same goes with special characters, for example:
- The "-" character in US keyboards is the " ' " character in spanish keyboards.
- The "=" character in US keyboards is the " ¡ " character in spanish keyboards.
- The "~" character in US keyboards is the " \ " character in spanish keyboards. (Although it actually is the " º " character, but you must bind the former).

I'd disagree this isn't a bug with the app, as long as it doesn't work with the base product it was designed for it's a bug. Valve won't adapt to your website so your bindings are cool, you are actually expected otherwise.

Besides, it shouldn't be hard to patch, I expect the cfg would be generated from a hardcoded template.

**Comment by cyberhck** [comment]
I don't know wtf valve did just recently, my KP_SLASH was working before, but since today it stopped working,

and I tried figuring out, came to this post, as a programmer, I really wanted to make my old bind work.

I tried in-game settings and while I pressed that key, it showed "/" instead of "KP_SLASH" and for some reason, that worked.

so I went into my autoexec and changed "KP_SLASH" to "/" and it started working again.

I don't know if valve fucks it up again, but if it does, the first thing I'll do it remember how I fixed it before (hello future me, did you notice the purple link in google search results? nice! :+1: ), anyway, I'll simply check if KP_SLASH will work later, (for everyone of you try to see if "/" works instead)
